com.unity.perception/Runtime/GroundTruth/SimulationState.cs

```diff
@@ -447,6 +447,16 @@ public void Update()
             {
                 var sensorData = m_Sensors[activeSensor];
 
+#if UNITY_EDITOR
+                if (UnityEditor.EditorApplication.isPaused)
+                {
+                    //When the user clicks the 'step' button in the editor, frames will always progress at .02 seconds per step.
+                    //In this case, just run all sensors each frame to allow for debugging
+                    sensorData.sequenceTimeOfNextRender = UnscaledSequenceTime;
+                    sensorData.sequenceTimeOfNextCapture = UnscaledSequenceTime;
```

Comment:
Besides the issues you mentioned in the PR description, this also causes a timing issue in multi-sensor simulations where the sensors have different delta times. This code will cause all sensors to capture at the same time regardless of their delta times. It would be nice if we could distinguish between pause and step, and only do this if step is used.

I can't think of an immediate solution though, and being able to use step is really important. So it may be worthwhile to just add some kind of warning message to the screen (or the log) when the player is paused, to inform the user that timing settings will not be respected if the simulation is paused?

Reply:
As far as I can tell, this code will only get hit if the user hits the step button. When the user pauses and unpauses it will never get hit. I could add an info message to the log mentioning this behavior - does that sound good?

Reply:
My bad, you're right it only gets called on step!
Yeah log message sounds good.

Reply:
Done!